src/transformState.js

```diff
@@ -63,6 +63,35 @@
  */
 function transformState(state, transforms) {
   // write code here
+  for (let i = 0; i < transforms.length; i++) {
+    const { operation, properties } = transforms[i];
+
+    if (operation === 'addProperties') {
+      const propertiesEntries = Object.entries(properties);
```

Comment:
you don't need to use Object.entries there, use `for (const prop in properties)`